doc/manual/de/install/update-other.rst

```diff
@@ -1,44 +1,44 @@
 .. replaces:: CometVisu/0.8.x/manual/updateother/de
 
-Update der CometVisu auf anderen Geräten
-========================================
+Update der direkt installierten CometVisu
+=========================================
 
 **Download der Installationsdatei**
 
 .. code-block:: bash
 
     cd /var/www
-    wget -O CometVisu0.9.2.zip  https://github.com/CometVisu/CometVisu/releases/download/v0.9.2/CometVisu-0.9.2.zip
+    wget -O CometVisu0.12.0.zip  https://github.com/CometVisu/CometVisu/releases/download/v0.12.0/CometVisu-0.12.0.zip
```

Comment:
Shall we really use real versions here? They need to be kept up to date manually. Something that we forget easily.

Reply:
That's something we also forgot in the past :)
I'd still keep it as an example - and in the hope that we aren't forgetting it too often.

Best would be a script that it automating that - but as we are releasing not so often, a script must also be kept running and an a bit outdated information here doesn't hurt I'd keep it as it is